### PR TITLE
[Enhancement] - Adding Create Ticket Function To Freshdesk

### PR DESCRIPTION
**Pull request by NirTatcher**
- Added a _post function for connector 
- Added a create ticket function that will be used to create a ticket in Freshdesk (more info here: https://developers.freshdesk.com/api/#tickets)

@shaunagm please let me know if there is any question or any feedback :)

### CODE REVIEW

File: parsons/freshdesk/freshdesk.py [hunk 1]
```diff
@@ -213,3 +230,39 @@ def get_agents(self, email=None, mobile=None, phone=None, state=None):
         tbl.remove_column("signature")  # Removing since raw HTML might cause issues.
 
         return tbl
+
+    def create_ticket(
+        self, subject, description, email, priority, status, cc_emails=None, custom_fields=None
+    ):
+        """
+        Create a ticket in Freshdesk.
```
Comment (shaunagm) [inline parsons/freshdesk/freshdesk.py:238]: Looks like cc_emails and custom_fields are optional, can you mark them as such in the docs with a "(optional)" next to the param name?

**Review by shaunagm — CHANGES_REQUESTED** [review]
One minor update to the docstrings and this should be good to merge.

File: parsons/freshdesk/freshdesk.py (hunk 1 above)
Comment (NirTatcher) [reply parsons/freshdesk/freshdesk.py:238]: Hi @shaunagm ofc!
done.
Thank you so much for reviewing and for the feedback!

Comment (shaunagm) [reply parsons/freshdesk/freshdesk.py:238]: Thank you!

**Review by shaunagm — APPROVED** [review]

**Pull request merged**